lec07-pop-models.Rmd

```diff
@@ -1,4 +1,617 @@
 ---
-title: "Qualitative population models in 2 dimensions"
+title: "Population models in two dimensions and numerically solving equations in R"
+author: "Madeleine Bonsma-Fisher"
+output: html_document
```

Comment:
You don't need author and output. Though, it probably would be a good idea to list who created which lessons.... Keep author in!